framework/transport/ieee1905_transport/test/ieee1905_transport_test.cpp

```diff
@@ -336,8 +336,8 @@ int main(int argc, char *argv[])
             std::copy_n(ieee1905_header, sizeof(ieee1905_header), cmdu_tx_msg.data());
 
             MAPF_INFO("sending CmduTxMessage: " << std::endl << cmdu_tx_msg);
-            rc = bus.publisher().Send(cmdu_tx_msg);
-            if (rc != true) {
+            bool ret = bus.publisher().Send(cmdu_tx_msg);
+            if (!ret) {
```

Comment:
I personally have a slight preference for

```suggestion
            if (!bus.publisher().Send(cmdu_tx_msg)) {
```

but we use both patterns in the code at the moment, and this is what it was originally, so OK. Maybe we should choose and track it in CODING_STYLE.md though.